Formula/subversion.rb

```diff
@@ -54,7 +56,7 @@ class Subversion < Formula
   patch :DATA
 
   def install
-    ENV.prepend_path "PATH", Formula["python@3.8"].opt_libexec/"bin"
+    ENV.prepend_path "PATH", "/System/Library/Frameworks/Python.framework/Versions/2.7/bin"
```

Comment:
Should already be in the path and used?

Reply:
As there is no `depends_on python@2 / uses_from_macos python@2` at all in the dependency list, this is not in the path. So it needs to be added.

Reply:
`/usr/bin/python` isn't in the PATH by default?

Reply:
Right, my bad. I removed the line.